NTO/Automation/entities/ActionHandlerCapability.ttl

```diff
@@ -0,0 +1,29 @@
+@prefix owl:                    <http://www.w3.org/2002/07/owl#> .
+@prefix ogit.Automation:        <http://www.purl.org/ogit/Automation/> .
+@prefix ogit:                   <http://www.purl.org/ogit/> .
+@prefix rdfs:                   <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix dcterms:                <http://purl.org/dc/terms/> .
+
+ogit.Automation:ActionHandlerCapability
+    a rdfs:Class;
+    rdfs:subClassOf ogit:Entity;
+    rdfs:label "ActionHandlerCapability";
+    dcterms:description """Configuration node to dynamically connect Knowledge Pools or Rankings to a HIRO installation.""";
+    # For ranges, see http://dublincore.org/documents/dcmi-period/
+    dcterms:valid "start=2019-03-08;";
+    dcterms:creator "dr@arago.de";
+    ogit:scope "NTO";
+    ogit:parent "http://www.purl.org/ogit/Node";
+    ogit:mandatory-attributes (
```

Comment:
At least fields for the capability name and required/optional parameters should be well defined ogit attributes, we do not want to endorse usage of free attributes all over the place ;)